src/org/usfirst/frc/team4915/stronghold/commands/ArcadeDrive.java

```diff
@@ -1,14 +1,19 @@
 
 package org.usfirst.frc.team4915.stronghold.commands;
 
+import edu.wpi.first.wpilibj.Joystick;
 import edu.wpi.first.wpilibj.command.Command;
 import org.usfirst.frc.team4915.stronghold.Robot;
 
-public class ExampleCommand extends Command {
+public class ArcadeDrive extends Command {
```

Comment:
I wouldn't make commands/new classes by overwriting old ones - even examples. When you do this you remove the example from the repo, the example will be gone from everyone's repository.